validation-test/BuildSystem/extractsymbols-no-file-to-process.test

```diff
@@ -0,0 +1,18 @@
+# COM: Ensure the logic for extracting symbols does not fail is there is nothing to process
```

Comment:
Can you remove the COM: prefix. I think it will be clear that it is a comment.

Reply:
Those are now removed.